libminifi/test/unit/IdTests.cpp

```diff
@@ -77,7 +77,7 @@ TEST_CASE("Test Generate Move", "[id]") {
 
   auto generated = generator->generate();
   auto str = generated.to_string();
-  utils::Identifier moved = std::move(generated);
+  utils::Identifier moved = generated;
```

Comment:
This change is also removing an intentional move.

Reply:
Updated in 7d86623adbe9d1cf018a62e72536bce62cd4332d